docusaurus/docs/guides/deployments/20-docker/30-console.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Console Deployment
+sidebar_label: Console
+---
+
+This article is about deploying the Ziti console as a standalone Node.js server application in a Docker container.
```

Comment:
I don't know why but I feel better not including the console in the whole "production install" sort of section. It feels like is should be in its own kind of section to me we already have much of this covered in the existing quickstart section. I'd feel better just removing it for now